nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/flow/change-version-dialog/change-version-dialog.html

```diff
@@ -22,32 +22,48 @@ <h2 mat-dialog-title>
     <mat-dialog-content>
         <div class="dialog-content flex flex-col gap-y-4 w-full">
             <div class="flex flex-row w-full">
-                <div class="flex flex-1 flex-col gap-y-4">
+                <div class="flex basis-1/2 flex-col gap-y-4 overflow-hidden">
```

Comment:
We had this issue in the create/edit connection templates. There is no padding in between the 'columns':
<img width="689" alt="Screenshot 2024-05-15 at 2 21 47 PM" src="https://github.com/apache/nifi/assets/6797571/0b68fab5-0b56-41c4-8f5d-f519efef4d08">

The fix was to use `w-1/2 pr-2` here